tests/test_cli.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+import subprocess
+import sys
+
+import pytest
+
+from trustme._cli import main
+
+
+def test_trustme_cli(tmpdir):
```

Comment:
Educational comment: pytest docs indicate that `tmpdir` is sorta deprecated and `tmppath` is the replacement so it should be used instead.